cmd/wfxctl/cmd/job/delete/delete.go

```diff
@@ -9,39 +9,34 @@ package delete
  */
 
 import (
-	"github.com/rs/zerolog/log"
+	"github.com/Southclaws/fault"
 	"github.com/spf13/cobra"
 
 	"github.com/siemens/wfx/cmd/wfxctl/errutil"
 	"github.com/siemens/wfx/cmd/wfxctl/flags"
-	"github.com/siemens/wfx/generated/client/jobs"
 )
 
-const (
-	idFlag = "id"
-)
-
-func init() {
-	f := Command.Flags()
-	f.String(idFlag, "", "id of the job which shall be deleted")
-}
-
-var Command = &cobra.Command{
-	Use:              "delete",
-	Short:            "Delete an existing job",
-	Long:             `Delete an existing job`,
-	TraverseChildren: true,
-	Run: func(cmd *cobra.Command, _ []string) {
-		baseCmd := flags.NewBaseCmd()
-		client := errutil.Must(baseCmd.CreateHTTPClient())
-		params := jobs.NewDeleteJobsIDParams().
-			WithHTTPClient(client).
-			WithID(flags.Koanf.String(idFlag))
-
-		// no content
-		if _, err := baseCmd.CreateMgmtClient().Jobs.DeleteJobsID(params); err != nil {
-			errutil.ProcessErrorResponse(cmd.OutOrStderr(), err)
-			log.Fatal().Msg("Failed to delete job")
-		}
-	},
+func NewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:              "delete",
+		Short:            "Delete an existing job",
+		Long:             `Delete an existing job`,
+		Example:          `wfxctl job delete 8ea1e9d7-28e6-4f1f-b444-a8d2d1ad7618`,
```

Comment:
Hm, this is inconsistent as it doesn't require `–id=` and its ID is a GUID not a number

Reply:
Yes, actually the other examples are wrong: `--id` is always a UUID and not a number (anymore ;)